### DISCUSSION TITLE
Zone bounding box information

### DISCUSSION THREAD

**Discussion by JadWehbeh** [larics/icuas22_competition#41] [Q&A] [answered]
Hello,

The rules state that "Prior to a run, a team will know the coordinates of the take off spot and bounding boxes of zones 2 and 3, both in the motion capture reference frame." How will that information be provided?

Thank you.

Best,
Jad

**Comment by fpetric** [comment, marked as answer]
Hello @JadWehbeh, 

the take off spot of the UAV is basically given with the UAV pose topic, since you can access that information and it is expressed in the world coordinate frame, the pose of the UAV at the start is the takeoff spot. Since we plan on changing the starting position in evaluations, you can do the same on your end by changing the parameters of the `kopterworx.launch` file [here](https://github.com/larics/icuas22_competition/blob/4338ebeec8e108ce445b20f5dcc41822f3a7dc2c/startup/challenge/session.yml#L23) (currently only x coordinate is given, while y defaults to zero). It would be best not to specify z coordinate :)

with respect to zone boundaries, for the simulation phase these are the zone dimensions in meters (and they will be fixed and the same in all layouts):

- Zone1: x = [-12.5,-8], y = [-7.5,7.5], z = [0,4.5]
- Zone2: x = (-8, 1], y = [-7.5,7.5], z = [0,4.5]
- Zone3: x = (1, 12.5], y = [-7.5,7.5], z = [0,4.5]

You can use zone bounding boxes in any way you like. We will add the zone definition to the rulebok in the next update.

**Note that these are the dimensions of the space, the UAV cannot actually go to pose where x=-12.5, you need to account for the size of the UAV.**

**Reply by JadWehbeh** [reply]
Thank you!
